src/core/ad_ext.jl

```diff
@@ -0,0 +1,12 @@
+using Tracker, Distributions, PDMats
+
+# The code below introduces Tracker and Distributions to each other to
+# make them work well together. The code can be removed after that issue
+# is fixed on the Distributions side.
+
+PDMats.PDMat(ta::TrackedArray) = PDMat([e for e in ta])
```

Comment:
We should probably add a note that this solution is inefficient.

Reply:
OK. 

Another question, we can use `ta.data` to replace `[e for e in ta]`, but `ta.data` is an ordinary (not tracked) array of ordinary (not tracked) elements. Is it OK for our case?

Reply:
I don't think this is the correct fix. We should just teach `cholesky` how to operate on `TrackedArray` IMO. `ta.data` will drop the tracking which is important not to break the differentiation chain.

Reply:
I agree.